objectRecognitionSource/src/main/java/edu/ml/tensorflow/classifier/YOLOClassifier.java

```diff
@@ -111,8 +111,9 @@ private void calculateTopPredictions(final BoundingBox boundingBox, final Priori
             double confidenceInClass = argMax.getMaxValue() * boundingBox.getConfidence();
             if (confidenceInClass > threshold) {
                 predictionQueue.add(new Recognition(argMax.getIndex(), labels.get(argMax.getIndex()), (float) confidenceInClass,
-                        new BoxPosition((float) (boundingBox.getX() - boundingBox.getWidth() / 2),
-                                (float) (boundingBox.getY() - boundingBox.getHeight() / 2),
+                        /* BoxPosition() now takes center points */
+                        new BoxPosition((float) boundingBox.getX(),
+                                (float) boundingBox.getY(),
```

Comment:
Here, we use the center point to create the bounding box. YOLO gives us the center point -- we now pass that to create the `BoxPosition` and convert the `top`, `left`, `bottom`, and `right` from the center, height, and width (which are what is passed to this constructor).  So the code conventions internally changed, but the external interface (the JSON output) does not semantically change -- just has the center information added.